apps/home/components/related_artists_context/annotation.jade

```diff
@@ -0,0 +1 @@
+.abrv-follow-annotation.avant-garde-s-headline Based on #[a(href=based_on.href) #{based_on.name}]
```

Comment:
https://stackoverflow.com/questions/10953326/what-is-a-concise-way-to-create-inline-elements-in-jade